security/pkg/pki/ca/controller/secret_test.go

```diff
@@ -158,7 +158,7 @@ func TestSecretController(t *testing.T) {
 				Namespace:   "test-ns",
 			},
 		}
-		controller, err := NewSecretController(createFakeCA(), defaultTTL,
+		controller, err := NewSecretController(createFakeCA(), defaultTTL, "cluster.local",
```

Comment:
Could you also add a test case for custom domain name? Thanks.

Reply:
IIUC, the secret_test.go don't generate real certificates, with CN, SAN, etc., so I will add the test elsewhere in a later PR